roles/ora-host/tasks/main.yml

```diff
@@ -165,9 +165,7 @@
     name: "{{ item.group }}"
     gid: "{{ item.gid }}"
     state: present
-  with_items:
-    - "{{ oracle_groups }}"
-    - "{{ asm_groups }}"
+  with_items: "{{ oracle_groups + asm_groups if oracle_edition != 'FREE' else oracle_groups }}"
```

Comment:
Any thoughts about simply clearing out ASM-related variables in non-ASM installs, to avoid needing complex logic here?